fearless/Common/View/TableViewCell/SwipableTableViewCell.swift

```diff
@@ -0,0 +1,474 @@
+import Foundation
+import UIKit
+import SwiftUI
+
+protocol SwipeButtonProtocol: UIButton {
+    var type: SwipeButtonType { get }
+}
+
+enum SwipeButtonType: Int, CaseIterable {
```

Comment:
rename please to SwipableCellButtonType. SwipeButtonType is very common, possible duplicates across the app or in external libs